### PR TITLE
Fix german translations - administration Part 2

### PR DESCRIPTION
**Pull request by tinect**
### 1. Why is this change necessary?
Currently, there are some errors in German translations and languagefiles.
Additionally to #480 

See changes without whitespace changes: https://github.com/shopware/platform/pull/531/files?w=1

### 2. What does this change do, exactly?
Fixes German translations more admin-modules.
- misleading context to English source, including punctuation mark
- grammar
- spelling
- add missing translations
- fix translation in German translation for taxes (https://github.com/shopware/platform/pull/531/files#diff-c18fcfcd63bde3113b8f71472fff2cb8)

Sorry, currently I haven't that much time to check all modules in detail. But this will follow. Promised.

### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/module/sw-order/snippet/de-DE.json [hunk 1]
```diff
@@ -129,29 +129,29 @@
...
-      "labelSelectStateButton":"Status ändern und E-Mail versenden",
-      "headlineErrorStateChange":"Fehler",
-      "labelErrorStateChange":"Beim setzen des Status ist ein Fehler aufgetreten. Zusätzliche Informationen:",
-      "labelErrorNoAction":"Kein neuer Zustand gewählt.",
+      "labelLastEditedBy": "Zuletzt geändert von",
+      "labelSystemUser": "System",
+      "labelSelectStatePlaceholder": "Bitte wählen ...",
+      "labelSelectStateButton": "Status ändern und E-Mail versenden",
+      "headlineErrorStateChange": "Fehler",
+      "labelErrorStateChange": "Beim setzen des Status ist ein Fehler aufgetreten. Zusätzliche Informationen:",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-order/snippet/de-DE.json]:
```suggestion
      "labelErrorStateChange": "Beim Setzen des Status ist ein Fehler aufgetreten. Zusätzliche Informationen:",
```

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json [hunk 2]
```diff
@@ -132,7 +132,7 @@
     },
     "prices": {
       "cardTitlePriceRule": "Neue Preisregel",
-      "placeholderRule": "Bitte wählen ...",
+      "placeholderRule": "Bitte wählen",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]: Placeholder bitte immer mit den "..."

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json [hunk 3]
```diff
@@ -308,7 +308,10 @@
       "deleteButtonCancel": "Zurück",
       "deleteButtonDelete": "Löschen",
       "linkOpenStreamPreview": "Vorschau öffnen",
-      "emptyStateDescription": "Für dieses Produkt existiert noch keine Cross Selling Konfiguration."
+      "emptyStateDescription": "Für dieses Produkt existiert noch keine Cross Selling Konfiguration.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]:
```suggestion
      "emptyStateDescription": "Für dieses Produkt existiert noch keine Cross Selling-Konfiguration.",
```

File: src/Administration/Resources/app/administration/src/module/sw-profile/snippet/de-DE.json [hunk 4]
```diff
@@ -28,7 +28,10 @@
       "notificationSaveSuccessMessage": "Deine Profilinformationen wurden erfolgreich gespeichert.",
       "notificationSaveErrorMessage": "Deine Profilinformationen konnten nicht gespeichert werden.",
       "labelUploadAvatar": "Lade ein Profilbild hoch",
-      "labelUseAsAvatar": "Als Profilbild nutzen"
+      "labelUseAsAvatar": "Als Profilbild nutzen",
+      "placeholderNewPasswordConfirm": "Wiederhole Dein neues Passwort ...",
+      "placeholderNewPassword": "Gib Dein neues Passwort ein ...",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-profile/snippet/de-DE.json]:
```suggestion
      "placeholderNewPassword": "Gib ein neues Passwort ein ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-promotion/snippet/de-DE.json [hunk 5]
```diff
@@ -104,19 +104,19 @@
...
+              "labelNoCodesExist": "Keine individuellen Codes vorhanden",
               "columnCode": "Aktionscode",
               "columnRedeemed": "Eingelöst",
               "columnCustomer": "Kunde",
               "buttonDeleteCode": "Löschen",
               "buttonOpenCustomer": "Kunde öffnen",
               "alerts": {
                 "errorNoPatternTitle": "Fehlendes Muster",
-                "errorNoPatternMessage": "Bitte gib ein Muster für Deine Codes an.",
+                "errorNoPatternMessage": "Bitte gib ein Muster für Deine Codes an!",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-promotion/snippet/de-DE.json]:
```suggestion
                "errorNoPatternMessage": "Bitte gib ein Muster für Deine Codes an.",
```

File: src/Administration/Resources/app/administration/src/module/sw-promotion/snippet/de-DE.json [hunk 6]
```diff
@@ -104,19 +104,19 @@
...
                 "successGeneratedTitle": "Code-Erstellung abgeschlossen.",
-                "successGeneratedMessage": "{count} Codes wurden generiert.",
-                "successDeletedTitle": "Code gelöscht.",
+                "successGeneratedMessage": "{count} neue Codes wurden generiert!",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-promotion/snippet/de-DE.json]:
```suggestion
                "successGeneratedMessage": "{count} neue Codes wurden generiert.",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-country/snippet/de-DE.json [hunk 7]
```diff
@@ -36,9 +36,9 @@
       "labelShippingAvailable": "Versand verfügbar",
       "labelTaxFree": "Steuerfrei",
       "labelForceStateInRegistration": "Bundesland bei Registrierung erforderlich",
-      "placeholderName": "Namen eingeben ...",
+      "placeholderName": "Namen eingeben",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-country/snippet/de-DE.json]:
```suggestion
      "placeholderName": "Namen eingeben ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-country/snippet/de-DE.json [hunk 8]
```diff
@@ -36,9 +36,9 @@
...
       "placeholderPosition": "1",
-      "placeholderIso": "ISO-Code eingeben ...",
+      "placeholderIso": "ISO-Code eingeben",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-country/snippet/de-DE.json]:
```suggestion
      "placeholderIso": "ISO-Code eingeben ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/snippet/de-DE.json [hunk 9]
```diff
@@ -1,10 +1,10 @@
 {
   "sw-settings-custom-field": {
     "general": {
-      "description": "Zusatzfelder",
+      "description": "Zusatzfelder hier verwalten.",
       "mainMenuItemGeneral": "Zusatzfelder",
       "placeholderSearchBar": "Durchsuche alle Zusatzfelder ...",
-      "tooltipTechnicalName": "Der technische Name muss eindeutig sein und kann nach dem Speichern nicht mehr verändert werden"
+      "tooltipTechnicalName": "Der technische Name muss eindeutig sein und kann nicht mehr verändert werden."
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/snippet/de-DE.json]:
```suggestion
      "tooltipTechnicalName": "Der technische Name muss eindeutig sein und kann später nicht mehr verändert werden."
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-document/snippet/de-DE.json [hunk 10]
```diff
@@ -65,7 +65,7 @@
       "helpTextDisplayLineItems": "Bestellpositionen werden auf dem Dokument einzeln aufgelistet",
       "helpTextDisplayLineItemPosition": "Bestellpositionen werden durchnummeriert",
       "helpTextDisplayPageCount": "Jede Seite des Dokuments wird durchnummeriert",
-      "helpTextItemsPerPage": "Anzahl an Bestellpositionen pro Seite",
+      "helpTextItemsPerPage": "Anzahl an Positionen pro Seite ...",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-document/snippet/de-DE.json]:
```suggestion
      "helpTextItemsPerPage": "Anzahl der Positionen pro Seite.",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-language/snippet/de-DE.json [hunk 11]
```diff
@@ -33,8 +33,8 @@
       "labelLocale": "Lokalisierung",
       "labelParent": "Erben von",
       "labelIsoCode": "ISO Code",
-      "placeholderName": "Namen eingeben ...",
-      "placeholderPleaseSelect": "Bitte wählen ...",
+      "placeholderName": "Namen eingeben",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-language/snippet/de-DE.json]:
```suggestion
      "placeholderName": "Namen eingeben ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-language/snippet/de-DE.json [hunk 12]
```diff
@@ -33,8 +33,8 @@
...
+      "placeholderPleaseSelect": "Bitte wählen",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-language/snippet/de-DE.json]:
```suggestion
      "placeholderPleaseSelect": "Bitte wählen ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-mailer/snippet/de-DE.json [hunk 13]
```diff
@@ -4,14 +4,15 @@
       "mainMenuItemGeneral": "Mailer",
       "textHeadline": "Mailer",
       "buttonSave": "Speichern",
-      "helpText": "For more informations look at <a href='https://docs.shopware.com/en/shopware-6-en'>documentation</a>"
+      "helpText": "Für mehr Information, besuche die <a href='https://docs.shopware.com/de/shopware-6-de'>Dokumentation</a>"
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-mailer/snippet/de-DE.json]:
```suggestion
      "helpText": "Für mehr Informationen, besuche die <a href='https://docs.shopware.com/de/shopware-6-de'>Dokumentation</a>"
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-number-range/snippet/de-DE.json [hunk 14]
```diff
@@ -44,7 +44,7 @@
       "placeholderPrefix": "Präfix eingeben ...",
       "placeholderSuffix": "Suffix eingeben ...",
       "placeholderPattern": "Pattern eingeben ... \"prefix_{n}_suffix\"",
-      "placeholderStart": "Startnummer eingeben ...",
+      "placeholderStart": "Startnummer eingeben",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-number-range/snippet/de-DE.json]:
```suggestion
      "placeholderStart": "Startnummer eingeben ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-payment/snippet/de-DE.json [hunk 15]
```diff
@@ -35,9 +35,9 @@
         "labelUseAsLogo": "Als Logo benutzen"
       },
       "topRule": "Verfügbarkeitsregel",
-      "placeholderName": "Namen eingeben ...",
-      "placeholderPosition": "Position eingeben ...",
-      "placeholderDescription": "Beschreibung eingeben ...",
+      "placeholderName": "Namen eingeben",
+      "placeholderPosition": "Position eingeben",
+      "placeholderDescription": "Beschreibung eingeben",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-payment/snippet/de-DE.json]:
```suggestion
      "placeholderName": "Namen eingeben ...",
      "placeholderPosition": "Position eingeben ...",
      "placeholderDescription": "Beschreibung eingeben ...",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-shopware-updates/snippet/de-DE.json [hunk 16]
```diff
@@ -41,7 +42,7 @@
         "mysqlVersion": "Erforderliche MySQL Version: {minVersion}. Installierte MySQL Version: {currentVersion}.",
         "writeableCheckValid": "Die folgenden Verzeichnisse und Dateien sind beschreibbar:<br/>{checkedDirectories}",
         "writeableCheckFailed": "Die folgenden Verzeichnisse und Dateien sind beschreibbar:<br>{failedDirectories}",
-        "validShopwareLicense": "Die eingesetzte Shopware Version erlaubt ein Update",
+        "validShopwareLicense": "Deine aktuelles Shopware ist veraltet. Möglicherweise möchtest Du ein Update in Betracht ziehen.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-shopware-updates/snippet/de-DE.json]:
```suggestion
        "validShopwareLicense": "Deine Shopware-Version ist veraltet. Du solltest ein Update in Betracht ziehen.",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-shopware-updates/snippet/en-GB.json [hunk 17]
```diff
@@ -9,7 +9,7 @@
       "currentVersion": "current version:"
     },
     "cards": {
-      "info": "What's New",
+      "info": "What's New?",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-shopware-updates/snippet/en-GB.json]:
```suggestion
      "info": "What's new?",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/de-DE.json [hunk 18]
```diff
@@ -3,7 +3,7 @@
     "general": {
       "mainMenuItemGeneral": "Steuern",
       "taxRateSymbol": "&#037;",
-      "placeholderSearchBar": "Durchsuche alle Steuern ..."
+      "placeholderSearchBar": "Durchsuche Steuern"
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/de-DE.json]:
```suggestion
      "placeholderSearchBar": "Durchsuche Steuern ..."
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/de-DE.json [hunk 19]
```diff
@@ -30,22 +30,24 @@
       "buttonSave": "Speichern",
       "notificationErrorTitle": "Fehler",
       "notificationErrorMessage": "Fehler beim Speichern des Steuersatzes."
+    },
+    "taxRuleCard": {
+      "labelTaxRate": "Steuersatz",
+      "createStateDescription": "Speichere zuerst den Steuersatz, um weitere Länder hinzufügen zu können.",
+      "emptyStateDescription": "Füge weitere Steuerraten für verschiedene Länder hinzu.",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/de-DE.json]:
```suggestion
      "emptyStateDescription": "Füge weitere Steuersätze für verschiedene Länder hinzu.",
```

File: src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/en-GB.json [hunk 20]
```diff
@@ -36,7 +36,7 @@
       "labelAppliesOn": "Applies On",
       "labelCountryName": "Country",
       "cardTitle": "Countries",
-      "searchBarPlaceholder": "Search Countries",
+      "searchBarPlaceholder": "Search Countries...",
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/en-GB.json]:
```suggestion
      "searchBarPlaceholder": "Search countries...",
```

**Review by GitEvil — CHANGES_REQUESTED** [review]

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json (hunk 2 above)
Comment (tinect) [reply src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]: Alles klar. Ich passe english dann auch entsprechend an, okay?

Comment (marcelbrode) [reply src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]: Um eventuelle Arbeit direkt präventiv zu vermeiden: Im Deutschen mit Leerzeichen zwischen Wort und Ellipsis und im englischen nicht!

Im Englischen pflegen wir das aber auch so, ja :)

**Comment by tinect** [comment]
@GitEvil Ready :-)

File: src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/en-GB.json [hunk 21]
```diff
@@ -47,7 +47,7 @@
       "createStateDescription": "Save the tax rate first to be able to add more countries.",
       "modalTitleDelete": "Delete Country",
       "textDeleteConfirm": "Are you sure you want to delete the country \"{name}\"?",
-      "placeholderStates": "Add States"
+      "placeholderStates": "Add States..."
```
Comment (GitEvil) [inline src/Administration/Resources/app/administration/src/module/sw-settings-tax/snippet/en-GB.json]:
```suggestion
      "placeholderStates": "Add states..."
```

**Review by GitEvil — APPROVED** [review]

**Comment by pantrtxp** [comment]
Hi @tinect, you still need to accept the "Contributor License Agreement" before we can go on with this PR

**Comment by tinect** [comment]
@pantrtxp Now it's rebased to master.

**Review by GitEvil — APPROVED** [review]

**Review by GitEvil — APPROVED** [review]

**Comment by pantrtxp** [comment]
Thank you for your contribution :blue_heart: :tada:

**Pull request closed without merging**